ubyssey/static/src/styles/modules/widgets/_upcoming_events_horizontal.scss

```diff
@@ -0,0 +1,51 @@
+.c-widget-upcoming-events-horizontal {
+  background-color: #f5f5f5;
+  width: 100%;
+  margin: 15px 0;
+
+  box-sizing: border-box;
+  padding: 25px 50px;
```

Comment:
I don't think this padding is necessary for mobile screens -- it makes the events seem pretty squished